src/fullcalendar/eventSources/freeBusyBlockedForAllEventSource.js

```diff
@@ -153,8 +151,8 @@ export default function(organizer, attendees, resources) {
 }
 
 /**
- * @param slots
- * @param start
+ * @param {object} slots the slots
+ * @param {object} start the start
```

Comment:
`start` does not seem to be an object.

Reply:
Wasn't sure. Is it a string?

Reply:
I checked, it should be a simple `Date` object (comes from ICAL.js via calendar-js).

Reply:
Fixed.